docs/styling/host.md

```diff
@@ -2,7 +2,7 @@
 
 Templates and Slots bring with them a handful of CSS pseudo-classes and pseudo-elements to help with styling.
 
-The `:host` pseudo-class is a reference to the custom element itself, but you're unable   `:root` pseudo-class in HTML. 
+The `:host` pseudo-class is a reference to the custom element itself, but you're unable to use the `:root` pseudo-class in HTML.
```

Comment:
I hope this is the intended language. I had trouble finding information on using `:root` in a Web Component so I'll need to do some testing.